tsconfig.json

```diff
@@ -0,0 +1,23 @@
+{
+    "compilerOptions": {
+      "module": "commonjs",
+      "declaration": true,
+      "target": "es6",
+      "noImplicitAny": true,
+      "sourceMap": true,
+      "outDir": "dist",
+      "baseUrl": ".",
+      "paths": {
+        "*": ["node_modules/*"]
```

Comment:
my understanding is that this should be implicitly part of the lookup? is it not??

Reply:
you are correct